packages/app/src/dimension-mapper/DimensionMapper.module.css

```diff
@@ -1,5 +1,5 @@
 .mapper {
-  flex: none;
+  grid-area: dimMapper;
```

Comment:
Changing from a flexbox to a grid layout for `.visArea` so I can position the new UI element above the visualization. It saves me from adding wrappers in `NxImageContainer`.